### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'pickAlgorithm')

### DISCUSSION THREAD

**Discussion by LuckyArdhika** [tailwindlabs/tailwindcss#5817] [Show and tell]
Hi, i'm using npm v7.20.0 and I want to install the latest version of tailwindcss (2.2.15) but when I install it with the command "npm install tailwindcss" the error appears:

verbose stack TypeError: Cannot read properties of null (reading 'pickAlgorithm')
706 verbose stack     at Integrity.match (C:\Users\Ardhika\AppData\Roaming\npm\node_modules\npm\node_modules\ssri\index.js:234:24)
706 verbose stack     at CachePolicy.satisfies (C:\Users\Ardhika\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\cache\policy.js:113:49)
706 verbose stack     at Function.find (C:\Users\Ardhika\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\cache\entry.js:172:25)
706 verbose stack     at async cacheFetch (C:\Users\Ardhika\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\cache\index.js:8:17)
706 verbose stack     at async fetch (C:\Users\Ardhika\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\fetch.js:82:7)
707 verbose cwd D:\WEB\jasamu\jasamu
708 verbose Windows_NT 10.0.18363
709 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Ardhika\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "tailwindcss" "--save-dev"
710 verbose node v16.9.1
711 verbose npm  v7.20.0
712 error Cannot read properties of null (reading 'pickAlgorithm')

but it only appears when I install the latest version of tailwinds, when I try to install v1.9.6, no error occurs during installation. do you think this is a bug or my fault? if this is my fault please help. Thank you.

**Comment by debrajBarua** [comment]
I 4991 timing reifyNode:node_modules/rxjs Completed in 315729ms
4992 timing reifyNode:node_modules/@angular-devkit/architect/node_modules/rxjs Completed in 341282ms
4993 timing reifyNode:node_modules/@angular-devkit/schematics/node_modules/rxjs Completed in 341357ms
4994 timing reifyNode:node_modules/@angular-devkit/build-webpack/node_modules/rxjs Completed in 341712ms
4995 timing reifyNode:node_modules/@angular-devkit/build-angular/node_modules/rxjs Completed in 341771ms
4996 timing reifyNode:node_modules/@angular-devkit/core/node_modules/rxjs Completed in 342264ms
4997 timing reify:rollback:createSparse Completed in 325140ms
4998 timing reify:rollback:retireShallow Completed in 0ms
4999 timing command:install Completed in 832585ms
5000 verbose stack TypeError: Cannot read properties of null (reading 'pickAlgorithm')
5000 verbose stack     at Integrity.match (C:\Users\debraj.barua\AppData\Roaming\npm\node_modules\npm\node_modules\ssri\index.js:234:24)
5000 verbose stack     at CachePolicy.satisfies (C:\Users\debraj.barua\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\cache\policy.js:113:49)
5000 verbose stack     at Function.find (C:\Users\debraj.barua\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\cache\entry.js:172:25)
5000 verbose stack     at async cacheFetch (C:\Users\debraj.barua\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\cache\index.js:8:17)
5000 verbose stack     at async fetch (C:\Users\debraj.barua\AppData\Roaming\npm\node_modules\npm\node_modules\make-fetch-happen\lib\fetch.js:82:7)
5001 verbose cwd F:\practise\p\UI\HMS
5002 verbose Windows_NT 10.0.14393
5003 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\debraj.barua\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "--quiet"
5004 verbose node v16.13.0
5005 verbose npm  v7.24.2
5006 error Cannot read properties of null (reading 'pickAlgorithm')
5007 verbose exit 1

i am facing same types of problem too. any body know the solutions?

**Comment by AhmadAlHalabi** [comment]
i face the same issue, any help?
